data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -80,7 +80,6 @@ then
 		--etcd-ca=/assets/tls/etcd-ca-bundle.crt \
 		--etcd-ca-key=/assets/tls/etcd-signer.key \
 		--manifest-etcd-image="${MACHINE_CONFIG_ETCD_IMAGE}" \
-		--etcd-discovery-domain={{.ClusterDomain}} \
```

Comment:
I think we can drop this from the template input variables too here https://github.com/openshift/installer/blob/b53ac2429c0792c754b285ddf6c263728265abdb/pkg/asset/ignition/bootstrap/bootstrap.go#L55 ??

Reply:
Sure.